go/oasis-test-runner/scenario/e2e/runtime/history_reindex.go

```diff
@@ -69,12 +70,17 @@ func (sc *historyReindexImpl) Fixture() (*oasis.NetworkFixture, error) {
 		},
 	}
 
+	// Add additional client node, with random batch size.
 	if n := len(f.Clients); n != 1 {
 		return nil, fmt.Errorf("expected single client node, got %d", n)
 	}
+	batchSize := rand.IntN(100)
```

Comment:
```suggestion
	batchSize := rand.UIntN(100)
```
In general, we should avoid random numbers in tests. 

Suppose reindexing doesn't work for N=10, but works for all other numbers. In this case:
- We will merge these changes only to figure out few days or weeks latter that this doesn't work for N=10.
- Once we see that test failed, we cannot run it locally because the test will probably not fail again. So in order to debug it, we first need to change the test and hardcode batch size, fix the bug, and finally undo changes to test.

Now suppose we allow random variables and you have 10 of them in one test. Will that be easy to debug and when will the test fail?

And as I said before, for me, it doesn't make sense to test if reindex works for different batch sizes. If it works for 10, it should also work for 20. If you do test this, you should test edge cases, e.g. 0, 1, 10, and infinity. But I would not do that, not in E2E test. And if you do that, `client.ConfigureRuntime` function doesn't help you anymore, it would be better to have an option to clear the runtime state.